### ISSUE TITLE
Commit "Mask out private packages." broke library import

### ISSUE THREAD

**Issue by AdrienCorenflos** [google-research/torchsde#3]
See this comment:

https://github.com/google-research/torchsde/commit/41fb567ae050ef6a178b452467ed64340c6cad16#r41181450

**Comment by lxuechen** [comment]
Hi,

I just tried installing the library via ssh from master, and this doesn't appear to be a problem. Could you provide more context? 

Thanks,
Chen

**Comment by AdrienCorenflos** [comment]
Hi,

If I simply do import torchsde, it fails with an error stating that there
is no _core module.

I will provide a colab replication tomorrow first thing in the morning if
that is not enough information.

Adrien

**Comment by lxuechen** [comment]
Does this error still occur when installing the master branch? Thanks.

**Comment by AdrienCorenflos** [comment]
Can't say right now as I don't have access to a pc, but I did install from
master on colab using pip install git+etc.

**Comment by lxuechen** [comment]
I just reproduced this on colab. I think it might have something to do with colab, since I'm not seeing this issue on my local machine or remote vm. 

I'll get this fixed ASAP. Thanks again for the report!

**Comment by lxuechen** [comment]
Just debugged this and found it to only be an issue for Python <= 3.6. Here's the related bug fix discussion https://bugs.python.org/issue30024

Will send in a commit soon to fix it.
